### DISCUSSION TITLE
7.3.1版本中如何引用ArXiv文献？

### DISCUSSION THREAD

**Discussion by jiawei-wu96** [tuna/thuthesis#817] [Q&A] [answered]
当我使用7.2.3版本编译时，引用如下bib文献
```
@Article{Burnashev1997,
  author  = {Burnashev, M. V. and Holevo, A. S.},
  journal = {arXiv preprint quant-ph/9703013},
  title   = {On Reliability Function of Quantum Communication Channel},
  year    = {1997},
}
```
可以正常生成所有要素：
Burnashev M V, Holevo A S. On reliability function of quantum communication channel[J].
arXiv preprint quant-ph/9703013, 1997

但是当我切换为7.3.1版本时，生成的参考文献变成了： 
Burnashev M V, Holevo A S. On reliability function of quantum communication channel[A]. 1997

主要变化为：文献类型从[J]变成了[A]，journal那一项消失了。

经过我的反复测试，发现当journal中含有 arXiv字样时，一律会变成后者。但只要去掉arXiv这个词，journal这一项就能正常显示。7.3.1版本似乎专门针对arXiv作出了改变？

**Comment by zepinglee** [comment, marked as answer]
> 7.3.1版本似乎专门针对arXiv作出了改变？

是的，参考 <https://github.com/zepinglee/gbt7714-bibtex-style/issues/89#issuecomment-950295152>。

总之，预印本应该跟期刊文章区分开。有些使用 `@article` 和 `journal` 主要是为了兼容一些过时的 bibtex style。

**Reply by jiawei-wu96** [reply]
感谢解答
